src/App.js

```diff
@@ -4,6 +4,7 @@ import FirstTest from './Components/FirstTest';
 import CustomAccordion from './Components/CustomAccordion';
 import ReactStrapAccordion from './Components/ReactstrapAccordion';
 import ReactBootstrapAccordion from './Components/ReactBootstrapAccordion';
+import MUIAccordion from './Components/MUIAccordion';
```

Comment:
i think this should be called `CatAccordion`

Reply:
No way.  I vote for KittenAccordion!